tmt/convert.py

```diff
@@ -109,47 +110,80 @@ def read(path, makefile, nitrate, purpose):
         if not testcases:
             raise ConvertError("No testcase found for '{0}'.".format(test))
         elif len(testcases) > 1:
-            log.warn("Multiple test cases found for '{0}', using {1}".format(
-                test, testcases[0].identifier))
-        testcase = testcases[0]
-        echo("test case found '{0}'.".format(testcase.identifier))
-        # Contact
-        if testcase.tester:
-            data['contact'] = '{} <{}>'.format(
-                testcase.tester.name, testcase.tester.email)
-            echo(style('contact: ', fg='green') + data['contact'])
-        # Environment
-        if testcase.arguments:
-            data['environment'] = tmt.utils.variables_to_dictionary(
-                testcase.arguments)
-            echo(style('environment:', fg='green'))
-            echo(pprint.pformat(data['environment']))
-        # Relevancy
-        field = tmt.utils.StructuredField(testcase.notes)
-        data['relevancy'] = field.get('relevancy')
-        echo(style('relevancy:', fg='green'))
-        echo(data['relevancy'].rstrip('\n'))
+            log.warn("Multiple test cases found for '{0}'.".format(
+                test))
```

Comment:
Shall we still give warning when multiple test cases found? Or just give and info message?

Reply:
info seems better

Reply:
Although `logger.info()` only displays with higher level of `DEBUG`.
Is there a logger or other function which would display by default or do you think this information shouldn't be seen unless you are running higherr `DEBUG`?

Reply:
Under `tmt run` we're using the method `info()` for such output. We could somehow enable it here as well. See also `pydoc tmt.utils.Common` for more info about the common class.

Reply:
Seems that it depends on the workdir functionality so it would need some adjustments...